src/pages/Beta/Stake/DetailModal/StakeWidget/index.tsx

```diff
@@ -101,14 +100,15 @@ const StakeWidget: React.FC<Props> = ({ stakingInfo, onClose, isRewardStake }) =
               />
 
               <Box>
-                <Percentage
-                  onChangePercentage={value => {
+                <NumberOptions
+                  onChange={value => {
                     setStepIndex(value)
                     onChangePercentage(value * 25)
```

Comment:
i think there is something fishy here at line `onChangePercentage(value * 25)` because value will be 25 already, in that case do we need to multiply it ? can you please confirm once, I might be missing something

Reply:
it is correct to multiply by 25, because when it is in the step variation the values are [1,2,3,4], 1*25 = 25..., besides that the values come from the hook useDerivedStakingProcess and in it you can see that it is set to 4
https://github.com/pangolindex/interface/blob/e70bd5aa2e690aafaa0b60596c73809e1d3b3720/src/state/stake/hooks.ts#L1231

Reply:
When it is box variation the values are [25,50,75,100] so we don't need to multiply by 25 in the box variation
https://github.com/pangolindex/components/blob/main/src/components/NumberOptions/NumberOptions.tsx#L9

Reply:
When it is box variation the values are [25,50,75,100] so we don't need to multiply by 25 in the box variation
https://github.com/pangolindex/components/blob/0f97b3a2b51ffebb82e8a2f2f9899f9e30bcd599/src/components/NumberOptions/NumberOptions.tsx#L9